services/core-web/src/components/navigation/NotificationDrawer.tsx

```diff
@@ -222,7 +227,7 @@ const NotificationDrawer: FC<INotificationDrawerProps> = (props) => {
                     )}
                     <Col>
                       <Typography.Text className="notification-info-text">
-                        {formatDateTime(activity.create_timestamp)}
+                        {formatDateTimeTz(activity.create_timestamp)}
```

Comment:
This is probably minor, but I don't think formatDateTimeTz is quite the right function here (it's intended for user inputted dates where we have the timezone as a separate data field so it gets passed in, and falls back on PT as a default). In this case we don't have a timezone to pass in but since it's a system-generated timestamp we would want to show it in the user's timezone.

I would suggest making yet another formatting function. I tested this one out and it worked on the incident table (same except it'll attempt to guess the user tz and default to PT if it fails. The guess function caches its guess). It would also be possible to use native JS datetime since moment.tz is only required to _convert_ between timezones, I just find it much harder to deal with the options for .toLocaleString() [MDS doc on options](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat#options)
```
export const formatDateTimeUserTz = (
  dateTime: Date | string,
): string =>
  {
    return moment(dateTime, true).isValid() && moment.tz(dateTime, moment.tz.guess() ?? DEFAULT_TIMEZONE).format(DATETIME_TZ_FORMAT)};
```

Reply:
Added a new function as suggested.